lib/rubocop/cop/lint/deprecated_open_ssl_constant.rb

```diff
@@ -33,6 +33,7 @@ class DeprecatedOpenSSLConstant < Base
 
         MSG = 'Use `%<constant>s.%<method>s(%<replacement_args>s)` instead of `%<original>s`.'
 
+        RESTRICT_ON_SEND = %i[new digest].freeze
```

Comment:
This cop didn't specifically restrict before but I've checked and the only relevant class methods for `OpenSSL::Cipher` and `OpenSSL::Digest` are `new` and `digest`.